database.sql

```diff
@@ -0,0 +1,35 @@
+DROP TABLE IF EXISTS question_reponse;
```

Comment:
It's used in the presentation example, it definitely shouldn't be in the root of the project, we'll move it to the right folder.